bin/p2-bin2pod/main.go

```diff
@@ -130,25 +131,35 @@ func makeTar(workingDir string) (string, error) {
 	if err != nil {
 		return "", fmt.Errorf("Couldn't make a new working directory %s for tarring: %s", tarContents, err)
 	}
+
 	err = os.MkdirAll(path.Join(tarContents, "bin"), 0744)
 	if err != nil {
 		return "", fmt.Errorf("Couldn't make bin directory in %s: %s", tarContents, err)
 	}
 	launchablePath := path.Join(tarContents, "bin", "launch")
-	err = uri.URICopy(*executable, launchablePath)
+
+	url, err := url.Parse(*executable)
```

Comment:
kingpin provides a [URL()](https://godoc.org/gopkg.in/alecthomas/kingpin.v2#ArgClause.URL) argument type that will parse the URL for you.